users/views.py

```diff
@@ -1,3 +1,68 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+from django.core.exceptions import ValidationError
+
+from users.models     import User
+from users.validators import validate_email, validate_password
+from my_settings      import SECRET_KEY, ALGORITHM
+
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            username        = data.get('username')
+            email           = data.get('email')
+            password        = data.get('password')
+            phone_number    = data.get('phone_number')
+            address         = data.get('address')
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+            if validate_email(email)==None:
+                return JsonResponse({'messasge':'INVALID EMAIL'}, status=400) 
+            if validate_password(password)==None:
+                return JsonResponse({'messasge':'INVALID PASSWORD'}, status=400) 
+            if User.objects.filter(email = email).exists():
+                return JsonResponse({'messasge':'E-MAIL ALREADY EXISTED'}, status=400)
+
+            User.objects.create(
+                username     = username,
+                email        = email,
+                password     = hashed_password,
+                phone_number = phone_number,
+                address      = address,
+                point        = 100000,
```

Comment:
```suggestion
                point        = DEFAULT_POINT,
```
기본 point를 상수화 시켜 코드 수정이 용이하게 끔 수정해 주세요.

Reply:
수정에 용이하게끔 따로 상수화 시켜주는거였군요! 감사합니다!